src/main/java/seedu/command/Help.java

```diff
@@ -13,30 +14,31 @@ public Help() {
     /**
      * This method allows the user to select the help message to
      * be displayed by type of command.
-     *
-     * @throws PacException Throws PacException when the user
-     *                       selects any command out of the list.
      */
-    public void selectHelpMessage() throws PacException {
+    public void selectHelpMessage() {
         help.printGetHelp();
         String typeOfHelp = help.getStringInput();
-        switch (typeOfHelp) {
-        case "1":
-            help.printStudentListHelp();
-            break;
-        case "2":
-            help.printEventHelp();
-            break;
-        case "3":
-            help.printCalendarHelp();
-            break;
-        case "4":
-            help.printAttendanceHelp();
-            break;
-        case "5":
-            help.printPerformanceHelp();
-            break;
-        default: throw new PacException("Invalid help function selected.");
+        while (!typeOfHelp.equals("back")) {
```

Comment:
For this new change, will the user be allowed to input "back" to exit back to the main part of PAC?

Reply:
yep, they input 'back' to go back to the 'home page'